recipes/_rsync.rb

```diff
@@ -30,10 +30,13 @@
 
 include_recipe 'omnibus::_compile'
 
+rsync_patch_file = File.expand_path("../../files/default/rsync.3.1.0.config.patch", __FILE__)
+
 remote_install 'rsync' do
   source 'ftp://ftp.netbsd.org/pub/pkgsrc/distfiles/rsync-3.1.0.tar.gz'
   version '3.1.0'
   checksum '81ca23f77fc9b957eb9845a6024f41af0ff0c619b7f38576887c63fa38e2394e'
+  patches [rsync_patch_file] if node["kernel"]["machine"] == "ppc64le"
```

Comment:
We really don't need `rsync` around any longer now that [Omnibus::FileSyncer](https://github.com/chef/omnibus/blob/master/lib/omnibus/file_syncer.rb) exists. I would rather we just nuke the `omnibus::_rsync` recipe.